components/ai_chat/core/common/mojom/settings_helper.mojom

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+module ai_chat.mojom;
+
+import "brave/components/ai_chat/core/common/mojom/ai_chat.mojom";
+
+struct ModelWithSubtitle {
+  Model model;
+  string subtitle;
+};
```

Comment:
We should also put GetModelsWithSubtitles on the main AIChat interface instead of building it in the frontend. Can refactor that in our main refactor follow-up

Reply:
moved